ads_deploy/docs.py

```diff
@@ -209,9 +209,7 @@ def get_jinja_filters() -> Dict[str, Callable]:
     saw_tsproj = set()
     name_cache = {}
 
-    @jinja2.evalcontextfilter
```

Comment:
Hang on, was this just not needed then? You've removed it and replaced it with nothing.

Reply:
You explained it more fully in https://github.com/pcdshub/pytmc/pull/280

Reply:
Yes, I did a poor job of explaining it here, but I mentioned in the other PR:
> We used this to wrap our filters but didn't use the context _(i.e., the one which the wrapper inserts into the filter)_, so I've just removed such references